pkg/ccl/backupccl/testdata/backup-restore/user-defined-functions

```diff
@@ -180,31 +180,14 @@ query-sql
 WITH descs AS (
   SHOW BACKUP LATEST IN 'nodelocal://0/test/'
 )
-SELECT database_name, parent_schema_name, object_name, object_type, is_full_cluster FROM descs
+SELECT
```

Comment:
why did this change?

Reply:
got rid of the tenant_settings system table. I also don't think the test needed to show all these other rows anyway.